packages/node/test/integration/get-state.spec.ts

```diff
@@ -20,6 +21,7 @@ describe("Node method follows spec - getAppInstances", () => {
   let storeService: IStoreService;
   let node: Node;
   let nodeConfig: NodeConfig;
+  let provider: Provider;
```

Comment:
```suggestion
  let provider: JsonRpcProvider;
```

Reply:
Why restrict the type of provider that can be passed in?